jetson-benchmarks/scripts/run-jetson-benchmarks

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+WORKDIR=$(mktemp -d)
+function on_exit() {
+    sudo rm -rf "$WORKDIR"
+}
+trap on_exit EXIT
+
+cp -r ${script_dir}/../. "$WORKDIR"
```

Comment:
Could you explain what's going on here?

Reply:
This script will be located at ${out}/bin, when invoked, it will copy everything in the $out dir to a temp folder and start the benchmark from there. We have to do this since the benchmark generates file so can't be run from a read-only context.

Reply:
Makes sense. Thanks!